recipes/pytransform3d/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pytransform3d" %}
+{% set version = "1.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 4608079a30ef3f8e927f3b3aee7b58a1e47cdcd7c46be6d889d1cb0f82650baa
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
can this recipe be noarch?

Reply:
Added `noarch: python` in 527351e! All dependencies are noarch and it's otherwise pure python, so it shouldn't be a problem.

Reply:
Dependencies can also be "arch'd". But if possible, all packages that can be should be noarch as it will work immediately on all platforms and the build-matrix is much smaller.